### DISCUSSION TITLE
Sync error

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [samschott/maestral#670] [Q&A] [answered]
I'm trying to rename a folder on my local repo, but I get this sync error:

![dbsync](https://user-images.githubusercontent.com/54762742/164019439-09db9f8f-ab8a-431e-9f1e-a90d61d2960c.png)

I've checked and there is no remote folder with the new name.

If it matters, the new name is the same of the old name, but lowercased.

**Comment by samschott** [comment, marked as answer]
As far as Dropbox is concerned, the casing is cosmetic only and both paths are the same. That being said, such moves typically still complete successfully, even though the [API docs](https://www.dropbox.com/developers/documentation/http/documentation#files-move) claim:

> Note that we do not currently support case-only renaming.

I even have an integration test which ensures that it works and it has been passing without issues:

https://github.com/samschott/maestral/blob/2f82cdc12e3eaccb90f9594cdf88f7137a27288b/tests/linked/integration/test_sync.py#L216

Are you seeing the error for the parent folder itself or one of its children? And how many items does it contain?

In the worst case, you might need to rename the folder on the Dropbox website instead.

**Reply by raffaelemancuso** [reply]
I have worked arround it by renaming from XXX to xxx_a and then back to xxx
